### ISSUE TITLE
Error in nextJS build -  "HTMLElement is not defined"

### ISSUE THREAD

**Issue by robhadfield** [GriffinJohnston/ldrs#31]
Hey! I'm trying to get the loader to work in my NextJS project but hitting a `500` error with `HTMLElement is not defined` on page load... 

Simple repro...
```
import { mirage } from 'ldrs';
import type {} from 'ldrs';
mirage.register();

export const PageOverlayLoader = () => {

  return (
    <div>
      <l-mirage></l-mirage>
    </div>
  );
};
```

The page loads A-OK with the `ldrs` component removed.

Any help appreciated!

**Comment by GriffinJohnston** [comment]
Did you try the steps in the NextJS guide? https://github.com/GriffinJohnston/ldrs/blob/main/framework-guides.md#nextjs

**Comment by robhadfield** [comment]
Hey @GriffinJohnston - thanks for the fast reply and apologies for my slow one!

Yes I did try those versions and it does work but the lag of loading dynamically seems noticeable even using a production version locally. I might see if the performance/cache is better on a prod server and report back :) 

I'll close this for now as it does work (thanks!) - I was just hoping for something pre-compiled to reduce the wait time.

**Comment by GriffinJohnston** [comment]
@robhadfield If you're having lag problems with your particular use case I would recommend just using the raw html & css (it's ready to go, just copy/paste). In the future I plan to add back React components, which would also fix this issue.

**Comment by robhadfield** [comment]
Awesome work, thanks @GriffinJohnston 👌

**Comment by GriffinJohnston** [comment]
@robhadfield In case it's still relevant for you - React exports have been added back as of `v1.1.3`

**Comment by mateusloubach** [comment]
Hey, I didn't want to create an ```Issue``` for something so small. But I've been having an issue with Mirage loader. It doesn't display the color set to it. just stays "Black".

Anyone know what happens? I tried other loaders and color works. Just Mirage that doesn't.

**Comment by GriffinJohnston** [comment]
@mateusloubach go ahead and make a separate issue for this. I'm on my phone but I'll take a look at this in a bit.

**Comment by GriffinJohnston** [comment]
@mateusloubach Never mind - fixed in v1.1.7, just update your package and you should be good to go.

**Comment by mateusloubach** [comment]
awesome! appreciate it.
